src/main/java/org/example/App.java

```diff
@@ -1,13 +1,83 @@
 package org.example;
 
-/**
- * Hello world!
- *
- */
-public class App 
-{
-    public static void main( String[] args )
-    {
-        System.out.println( "Hello World!" );
+import org.example.model.Product;
+import org.example.model.ProductRepository;
+import org.example.model.ProductVersion;
+import org.example.model.ProductVersionRepository;
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.Configuration;
+
+import java.util.List;
+
+public class App {
+
+    public static void main(String[] args) {
+        Configuration con = new Configuration().configure();
```

Comment:
"config" would be better name, there is no apparent advantages in having less letters, especially when the readability is lower.

Reply:
changed name "configuration"